lib/fastifySession.js

```diff
@@ -62,13 +62,14 @@ function decryptSession (sessionId, options, request, done) {
         newSession(secret, request, cookieOpts, idGenerator, done)
         return
       }
-      if (session?.expires && session.expires <= Date.now()) {
+      if (session?.cookie?.expires && session.cookie.expires <= new Date()) {
         const restoredSession = Session.restore(
```

Comment:
We should re-evaluate:
- Whether or not to use object params. Once we get to like 8 it gets pretty confusing
- Whether Session.restore should exist at all. At this point, it's mostly a wrapper around prevSession